lib/transaction/transaction.js

```diff
@@ -279,7 +289,8 @@ Transaction.prototype.toBuffer = function() {
 };
 
 Transaction.prototype.toBufferWriter = function(writer) {
```

Comment:
Discussed that we have to throw an exception here. It's pretty complicated because of old tests, so we decided to add `@private` JSDoc tag to this method.